pages/api/check-status.ts

```diff
@@ -63,24 +68,27 @@ export const searchPassportStatusApi = async (
   const response = await fetch(url)
 
   if (response.ok) {
+    logger.info('response OK')
```

Comment:
This should be logged as `debug`.. we're gonna have thousands (perhaps millions) of these lines in the log file if this is left as `info`.